gaia-fast-list.js

```diff
@@ -865,6 +917,63 @@ Internal.prototype = {
     img.style.opacity = 0;
   },
 
+  normalizeImageSrc(src) {
+    if (typeof src == 'string') return src;
+    else if (src instanceof Blob) return URL.createObjectURL(src);
+    else throw new Error('image src invalid');
+  },
+
+  getCachedImage(index) {
```

Comment:
bikeshed: I wonder if the naming should reflect that this is an in-memory cache (as opposed to the localStorage dom cache feature). `getMemoryCachedImage`/`cacheImageInMemory`?
(don't bother if the name gets too ugly :))